spec/controllers/systems_spec.cr

```diff
@@ -364,58 +395,58 @@ module PlaceOS::Api
         end
       end
 
-      # TODO: awaiting addition of `core` to test environment
-      pending "GET /:sys_id/functions/:module_slug" do
-        cs = Model::Generator.control_system.save!
+      describe "with core" do
```

Comment:
For each spec that calls `core`, pls add a `tags: ["core"]` argument to the `it`

That way we can run specs without  core